docs/source/streamflow-public-testnet.md

```diff
@@ -0,0 +1,312 @@
+
+- The value received per hour (in terms of ticket expected value)
+- The frequency of winning tickets (in terms of hours)
+
+### Testing
+
+You can test your orchestrator setup by setting up your own broadcaster and routing the broadcaster's requests directly to your orchestrator.
+
+Start a broadcaster that will connect directly to your orhcestrator:
+
+```
+$ livepeer -network rinkeby -broadcaster -orchAddr <ORCH_SERVICE_URI>
+```
+
+`<ORCH_SERVICE_URI>` should be the publicly accessible `serviceURI` that your orchestrator registered on-chain.
+
+Follow the steps in the [broadcasting](#broadcasting) section to deposit funds and configure broadcasting preferences.
+
+Finally, stream video into your broadcaster and you should see your orchestrator start to transcode the incoming video.
+
+### Scaling transcoding
+
+The easiest way to setup an orchestrator to transcode video is to have the orchestrator perform solo transcoding by passing in both the `-orchestrator` and `-transcoder` flags when running `livepeer`. However, if you want to scale out your transcoding operation to support many concurrent streams, you will want to enable split transcoding.
+
+Split transcoding consists of running many individual transcoder nodes (usually on separate remote machines) that connect to your orchestrator. When your orchestrator receives video from a broadcaster, it will then distribute the incoming segments to the invidual transcoders instead of transcoding the video on its own. An orchestrator can increase its capacity and scale horizontally by adding more transcoders to its backend.
+
+In order to enable split transcoding, you should first run an orchestrator with solo transcoding turned off:
+
+```
+$ livepeer -network rinkeby -orchestrator -pricePerUnit 1 -orchSecret <ORCH_SECRET>
+```
+
+`<ORCH_SECRET>` is a secret defined by the orchestrator that is used to authenticate requests from transcoders. The secret should be shared with all transcoders to be attached to the orchestrator.
+
+Next, you should attach a transcoder to your orchestrator:
+
+```
+$ livepeer -transcoder -orchAddr <ORCH_SERVICE_URI> -orchSecret <ORCH_SECRET>
+```
+
+`<ORCH_SECRET>` should be the secret defined by your orchestrator. `<ORCH_SERVICE_URI>` should be the publicly accessible `serviceURI` that your orchestrator registered on-chain.
+
+You can run the above command on any number of machines that you would like to dedicate to transcoding.
+
+### GPU transcoding
+
+When you setup split transcoding for your orchestrator, you can run individual transcoders on not only CPUs, but GPUs as well. The GPUs that transcoders run on can be dedicated purely to transcoding or can also be mining cryptocurrencies simultaneously. See the [following guide](https://github.com/mk-livepeer/bot-miner-setup) for instructions on how to setup transcoders on GPUs.
```

Comment:
I think this section is a fine placeholder/pointer, but I'll probably add a little more context in a separate commit.